mkudffs/mkudffs.c

```diff
@@ -403,6 +403,11 @@ void split_space(struct udf_disc *disc)
 				size = ext->blocks;
 			}
 
+			// Make sure partition doesn't have a space bitmap with a partial last byte.
+			// It's a corner case that some systems (Windows chkdsk) don't handle properly.
+			if ((disc->flags & FLAG_UNALLOC_BITMAP) && (size & 7))
+				size &= ~7;
```

Comment:
Is this problem only for unallocated bitmap? Or also with freed bitmap?

Reply:
Yes; when I try to run chkdsk on a partition formatted with --space=freedbitmap I get this:

`Only unallocated space bitmap is supported for free space
management in UDF. Chkdsk aborted.`

Reply:
Ok. I'm fine with this one commit. But github UI does not allow me to merge just one commit...

Reply:
So move other commit into separate pull request and so I could merge this one.

Reply:
I cherry-picked manually this one commit into master branch.